internal/datastore/spanner/options.go

```diff
@@ -17,9 +17,20 @@ type spannerOptions struct {
 	disableStats                bool
 	readMaxOpen                 int
 	writeMaxOpen                int
-	schemaWatchHeartbeat        time.Duration
 	minSessions                 uint64
 	maxSessions                 uint64
+	migrationPhase              string
+}
+
+type migrationPhase uint8
+
+const (
+	keepExistingChangestreams migrationPhase = iota
```

Comment:
I understand we are not using this to control any behavior in SpiceDB - it's just in place so the operator can set a flag that does not cause SpiceDB to crash due to an unrecognized flag.

Reply:
Fixed